website/static/js/osfLanguage.js

```diff
@@ -70,5 +73,13 @@ module.exports = {
                 'associated with this account.',
             deauthError: 'Could not disconnect the Google Drive account at this time',
         }
+    },
+    apiOauth2Application: {
+        discardUnchanged: 'Are you sure you want to discard your unsaved changes?',
+        deactivateConfirm: 'Are you sure you want to deactivate this application for all users and revoke all access tokens? This cannot be reversed.',
+        deactivateError: 'Could not delete application. ' + REFRESH_OR_SUPPORT,
+        dataFetchError: 'Data not loaded. ' + REFRESH_OR_SUPPORT,
```

Comment:
This error message is ambiguous:

![osf___developer_applications](https://cloud.githubusercontent.com/assets/2379650/9584078/058a01f6-4fdd-11e5-8275-63aa35bef630.png)

How about "Could not load Developer Applications at this time. Please refresh ..."